packages/buidler-core/src/builtin-tasks/node.ts

```diff
@@ -18,10 +24,13 @@ import {
   ResolvedBuidlerConfig,
 } from "../types";
 
-import { TASK_NODE } from "./task-names";
+import { TASK_NODE, TASK_NODE_WATCH_COMPILER_OUTPUT } from "./task-names";
 
 const log = debug("buidler:core:tasks:node");
 
+const sleep = (timeout: number) =>
```

Comment:
This isn't used, isn't it? I guess it was here for debugging purposes.

Reply:
Oops, yeah. I was using this before finding out about `awaitWriteFinish`. Thanks!